rcl_yaml_param_parser/src/parser.c

```diff
@@ -1345,7 +1345,9 @@ static rcutils_ret_t parse_key(
             break;
           }
 
-          ret = rem_name_from_ns(ns_tracker, NS_TYPE_NODE, allocator);
+	  allocator.deallocate(node_name_ns, allocator.state);
+
+	  ret = rem_name_from_ns(ns_tracker, NS_TYPE_NODE, allocator);
```

Comment:
There's some weird indentation happening here; probably tabs instead of spaces.

Reply:
Thanks. I fix it (bca8dad79f057385a5756e0ee28a3d45ce447478)